arch/aarch64/thread.c

```diff
@@ -1,5 +1,11 @@
 #include <arch/thread.h>
 
+/////////////////////////////////////////////////////////////////////////////
+// \breif the function convers task state in a process control block, changes
```

Comment:
Typo: `breif` => `brief` (seen elsewhere too)

Reply:
Uh, thanks for noticing.